File: Fikra.Ng/wwwroot/fikra/src/app/models/effort.ts
export class Effort{
    id: string;
    createdon: Date;
    modifiedon: Date;
    estimated: number;
    remaining: number;
    completed: number;

    constructor(){
        this.createdon = new Date();
        this.modifiedon = new Date();
        this.estimated = 0;
        this.remaining = 0;
        this.completed = 0;
    }
}

File: Fikra.Model/Entities/Effort.cs
using Fikra.Model.Interfaces;
using System;
using System.Collections.Generic;
using System.Text;

namespace Fikra.Model.Entities
{
    public class Effort : IEntity<Guid>
    {
        public Guid Id { get; set; }
        public DateTime CreatedOn { get; set; }
        public DateTime ModifiedOn { get; set; }
        public bool IsDeleted { get; set; }
		public double Estimated { get; set; }
        public double Remaining { get; set; }
        public double Completed { get; set; }
    }
}
